Replace version command with --version option

FrNecas opened the pull request:
Fixes: #375 

Do you guys agree with the position in help? I think it makes the most sense to have it next to `--help` option.

FrNecas commented:
Sorry, missed the test for version, will have a look at it and fix it ;)

lachmanfrantisek commented:
> Sorry, missed the test for version, will have a look at it and fix it ;)

np, thanks for the fix.

FrNecas commented:
Tests are passing now, I removed the direct test since I don't see any way to achieve the same thing with using `click.version_option`. Also fixed the TODO.
